src/ig/IGExporter.ts

```diff
@@ -251,7 +251,38 @@ export class IGExporter {
     const inputIndexMarkdownPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.md');
     const inputIndexXMLPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.xml');
     let generation: ImplementationGuideDefinitionPageGeneration = 'markdown';
-    if (existsSync(inputIndexMarkdownPageContentPath)) {
+    let filePath: string;
+    if (existsSync(inputIndexMarkdownPageContentPath)) filePath = inputIndexMarkdownPageContentPath;
+    if (existsSync(inputIndexXMLPageContentPath)) filePath = inputIndexXMLPageContentPath;
+    if (existsSync(inputIndexMarkdownPagesPath)) filePath = inputIndexMarkdownPagesPath;
+    if (existsSync(inputIndexXMLPagesPath)) filePath = inputIndexXMLPagesPath;
+
+    if (this.config.indexPageContent) {
+      ensureDirSync(pageContentExportPath);
```

Comment:
So we prefer to use the `pagecontent` folder instead of `pages`, is that correct? What is the reasoning for that?

Reply:
Yes, that's correct. But I don't have much of a reasoning beyond that is where we were putting the generated file before, which really isn't much of a great answer..

Reply:
That's good enough for me though.